lib/EnsEMBL/REST/Model/LDFeatureContainer.pm

```diff
@@ -81,6 +81,41 @@ sub fetch_LDFeatureContainer_variation_name {
   return $self->to_array($ldfc);
 }
 
+sub fetch_LDFeatureContainer_slice {
+  my ($self, $slice) = @_;
+  Catalyst::Exception->throw("No region given. Please specify a region to retrieve from this service.") if ! $slice;
+  Catalyst::Exception->throw("Specified region is too large. Maximum allowed size for region is 1Mb.") if ($slice->length > 1_000_000);
+  my $c = $self->context();
+  my $species = $c->stash->{species};
+
+  my $ldfca = $c->model('Registry')->get_adaptor($species, 'Variation', 'LDFeatureContainer');
+
+  my $ld_config = $c->config->{'Model::LDFeatureContainer'};
+  if ($ld_config && $ld_config->{use_vcf}) {
+    $ldfca->db->use_vcf($ld_config->{use_vcf});
+    $Bio::EnsEMBL::Variation::DBSQL::VCFCollectionAdaptor::CONFIG_FILE = $ld_config->{vcf_config};
```

Comment:
I've questioned this elsewhere, but this kind of direct variable access is not OO and should be avoided if possible. It makes code untrustworthy and harder to test. It would be nicer if the VCFCollectionAdaptor had an accessor for this.